src/runtime/CMakeLists.txt

```diff
@@ -1,3 +1,9 @@
+find_package(
+    VulkanHeaders 1.3.296 REQUIRED
+    HINTS "${Halide_SOURCE_DIR}/dependencies/vulkan"
+)
+get_target_property(VulkanHeaders_INCLUDE_DIR Vulkan::Headers INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
On this subject, can we add `vulkan-headers` to `./vcpkg.json`?

Reply:
Will also need to update the `builtin-baseline` to `d567b667adba0e72c5c3931ddbe745b66aa34b73`

Reply:
@alexreinking Hrmm ... updating the builtin-baseline is generating an error on the windows builds. Is there another file I need to modify?

Reply:
We might need to `git pull` vcpkg on the bots.

Reply:
@derek-gerstmann - I updated vcpkg on win-worker-4 and the builds seem to be working

Reply:
Awesome, thank you!